autoload/coqtail/compat.vim

```diff
@@ -6,6 +6,7 @@ let g:coqtail#compat#t_dict = type({})
 let g:coqtail#compat#t_list = type([])
 let g:coqtail#compat#nvim = has('nvim')
 let g:coqtail#compat#has_channel = (has('channel') && has('patch-8.0.0001')) || g:coqtail#compat#nvim
+let g:coqtail#compat#has_deletebufline = exists('*deletebufline')
```

Comment:
Instead of this we could define a `coqtail#compat#deleteline(first, last)` function that calls `deletebufline('%', first, last)` if it can and otherwise does `silent first,last delete _`.